src/utils.js

```diff
@@ -0,0 +1,9 @@
+export default function matches(textToMatch, node, matcher) {
```

Comment:
Rather than exporting this as the default (because this file is called `utils`), could we make it a named export (and feel free to disable the eslint warning, that's another one I've been meaning to disable).

```javascript
function matches() {/* etc... */}

export {matches}
```